src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
```

Comment:
Please create a single instance of Random at the class level, rather than creating a new instance each time it's used. This helps improve performance and code readability. Example: private static final Random RANDOM = new Random();

Reply:
I changed field from "private static final Random RANDOM = new Random();" to "private final Random random = new Random();", because after execution command "mvn clean package" I got next error message: "Static variable definition in wrong order".
![Static RANDOM](https://user-images.githubusercontent.com/115174410/233036582-489de9cd-f077-495c-9299-494972255263.jpg)